src/test/java/com/google/step/datamanager/RestaurantManagerDatastoreTest.java

```diff
@@ -0,0 +1,111 @@
+package com.google.step.datamanager;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.model.Restaurant;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class RestaurantManagerDatastoreTest {
+
+  private static final String RESTAURANT_NAME_A = "A";
+  private static final String BLOBKEY_A = "a_blob_key";
+
+  private static final String RESTAURANT_NAME_B = "B";
+  private static final String BLOBKEY_B = "b_blob_key";
```

Comment:
these constants can be moved to TestConstants.java since it is merged into master already